build.py

```diff
@@ -1,7 +1,21 @@
+import copy
 from cpt.packager import ConanMultiPackager
 
 if __name__ == "__main__":
     builder = ConanMultiPackager(build_policy = "missing", archs = ["x86_64"], options = ["stlab:testing=True"])
         # apple_clang_versions= ["12.0"]
     builder.add_common_builds()
+
+    filtered_builds = []
+    for settings, options, env_vars, build_requires, reference in builder.items:
+    
+        if settings["compiler"] == "clang" and settings["build_type"] == "Release":
```

Comment:
For my understanding: Why only for build_type === Release?

Reply:
Oh good catch. I forgot to remove that.
It was a wrong requirement in the recent `libdispatch` Conan recipe.